app/models/concerns/smooch_capi.rb

```diff
@@ -4,6 +4,11 @@ module SmoochCapi
   extend ActiveSupport::Concern
 
   module ClassMethods
+    def should_ignore_capi_request?(request)
+      event = request.params.dig('entry', 0, 'changes', 0, 'value', 'statuses', 0, 'status').to_s
```

Comment:
just curious - will we ever hit this for a non-CAPI request? if so, would it make sense to check for the request origin specifically (like, if we have something that indicates origin in the params)?

I'm assuming the params will just be different based on the request origin and that's how we're protecting against non-CAPI request types, but wonder if there's a way to make clearer to us.

if not, I think the concern and method name indicates its use. I just have always found this pattern (indicating request origin by the method name we're calling, without trying to understand the request origin before calling the method) a little confusing

Reply:
Good question! Right, we will never hit that for non-CAPI requests because that structure is very specific. In other methods (for cases we handle), we also check for headers that indicate that the request is coming from CAPI.